src/components/Points/CampaignTabs.tsx

```diff
@@ -1,65 +1,110 @@
 import Tabs from '@mui/material/Tabs'
 import Tab from '@mui/material/Tab'
 import { Box, Chip, Tooltip, Typography } from '@mui/material'
-import { useTheme } from '@mui/material/styles'
 import css from './styles.module.css'
+import { Campaign, useCampaignPage } from '@/hooks/useCampaigns'
+import { ReactNode, useMemo } from 'react'
+import { useGlobalCampaignId } from '@/hooks/useGlobalCampaignId'
 
-const CAMPAIGN_TABS = [
-  {
-    label: (
-      <Typography display="flex" flexDirection="row" gap={1} alignItems="center" fontWeight={700}>
-        <Tooltip title="This campaign is active now" arrow>
-          <Box
-            sx={{
-              borderRadius: '100%',
-              backgroundColor: ({ palette }) => palette.primary.main,
-              minWidth: '6px',
-              minHeight: '6px',
-              flexShrink: 0,
-              marginRight: 1,
-            }}
-          />
-        </Tooltip>
-        Global
-      </Typography>
-    ),
-    disabled: false,
-  },
-  {
-    label: (
-      <Typography display="flex" flexDirection="row" gap={1} alignItems="center" fontWeight={700}>
+type CampaignTabProps = {
+  label: ReactNode
+  disabled: boolean
+  value: string
+}
+
+const CampaignTab = ({ campaign }: { campaign: Campaign }) => {
+  const hasStarted = new Date(campaign.startDate).getTime() < Date.now()
+  const hasEnded = new Date(campaign.endDate).getTime() < Date.now()
+  const isActive = hasStarted && !hasEnded
+  return (
+    <Typography display="flex" flexDirection="row" gap={1} alignItems="center" fontWeight={700}>
+      <Tooltip title={`This campaign is ${isActive ? 'active' : 'inactive'}`} arrow>
         <Box
           sx={{
             borderRadius: '100%',
-            backgroundColor: ({ palette }) => palette.text.disabled,
+            backgroundColor: ({ palette }) => (hasStarted && !hasEnded ? palette.primary.main : palette.text.disabled),
             minWidth: '6px',
             minHeight: '6px',
             flexShrink: 0,
             marginRight: 1,
           }}
         />
-        Campaigns <Chip variant="outlined" className={css.comingSoon} label="soon" />
-      </Typography>
-    ),
-    disabled: true,
-  },
-] as const
+      </Tooltip>
+      {campaign.name} {!hasStarted && <Chip variant="outlined" className={css.comingSoon} label="soon" />}
+    </Typography>
+  )
+}
+
+const CampaignTabs = ({
+  onChange,
+  selectedCampaignId,
+}: {
+  onChange: (campaignId: string) => void
+  selectedCampaignId: string
+}) => {
+  const campaigns = useCampaignPage(5)
```

Comment:
Are we ordering the campaigns in any way e.g. newest on top?

Reply:
Currently its not possible to sort them somehow. But it might be added to the backend.